### DISCUSSION TITLE
Encountered two children with the same key

### DISCUSSION THREAD

**Discussion by cmpscabral** [icflorescu/mantine-datatable#208] [Q&A]
Hi,

I keep getting this error, even when specifying idAccessor:

Warning: Encountered two children with the same key, `one_of_my_keys`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.

How can I avoid this?
Thank you!

**Comment by icflorescu** [comment]
The row keys are your record ids, so this shouldn't happen if your ids are indeed unique.
Can you please check if they are, and if they do, can you please provide a codesandbox showcasing what's happening?

Thanks!

**Comment by cmpscabral** [comment]
Thank you for replying - it was my fault indeed.

All the keys are unique, but I had two columns with the same accessor ID (one of them was a virtual column, so I didn't pay too much attention).

Thanks again!
